guides/source/association_basics.md

```diff
@@ -2724,3 +2724,156 @@ will run a query like:
 ```sql
 SELECT "vehicles".* FROM "vehicles" WHERE "vehicles"."type" IN ('Car')
 ```
+
+Delegated Types
+----------------
+
+[`Single Table Inheritance (STI)`](#single-table-inheritance-sti) works best when there is little difference between subclasses and their attributes, but includes all attributes of all subclasses I need to create a single table.
+
+This has the disadvantage that attributes that are only used by a specific subclass are added to the same table, even though they are unnecessary attributes for another subclass, which bloats a single table.
+
+In the following example, the `Comment` does not need a `subject`, but the `Message` need a `subject`, so the `Entry` includes the `subject` attribute.
+
+```ruby
+# Schema: entries[ id, type, subject, created_at, updated_at]
+class Entry < ApplicationRecord
+end
+
+class Comment < Entry
+end
+
+class Message < Entry
+end
+```
+
+`Delegated types` solves this problem.
```

Comment:
We generally only backtick stuff that is actual code.

```suggestion
Delegated types solves this problem, via `delegated_type`.
```

Reply:
OK, I removed the backtick in the following part.

https://github.com/rails/rails/pull/46190/files#diff-d977574b785ce95fe656c58211c668992bf95214d52d3ffadb409ef8f69ad287R2749